src/AddTodoForm.js

```diff
@@ -1,10 +1,19 @@
 import React from "react";
 
-function AddTodoForm () {
+function AddTodoForm(props) {
+    const [inputText, setInputText] = React.useState("");
+        function handleAddTodo(event) {
+            event.preventDefault(); //Prevents default behavior of the form submit
+            let todoTitle = inputText; //Retrives the value of title element from target and stores it in todoTitle
```

Comment:
Inside of the variable todoTitle, should be stored the value retrieved from the event object (this is passed as an argument in handleAddTodo function) instead of the inputText state . Example:

let todoTile = event.target.title.value

This would fix your code.

Reply:
Great thank you.